### DISCUSSION TITLE
Connect To MongoDB

### DISCUSSION THREAD

**Discussion by nice9uy** [Akryum/vue-apollo#869] [Q&A]
Hi..

I am beginner with vue-apollo what I ask is how do vue-apollo connect to mongodb server? 

![Screenshot from 2019-12-03 09-29-51](https://user-images.githubusercontent.com/25494182/70015302-87cc6180-15af-11ea-9e02-77727d71e087.png)

Which file and example script, please..

Thanks a lot

**Comment by Austio** [comment]
@nice9uy i don't think this is the right place for this question.  To help
 VueApollo - Provides Apollo to Vue
 Apollo - Connects to Graphql
 Graphql - (in your case) Acts as proxy between requests and your mongo database
 MongoDB - Is a Datastore

So what you need to do is find out how to expose your mongodb with graphql, then point apollo to your graphql url and you should be in business.
